labchecker/labcheck.py

```diff
@@ -1,25 +1,49 @@
 from plugins.base import Plugin
+from utils import _color
+import utils
 import sys
 import getopt
+import yaml
 
 
 def usage():
-    print("Usage: labchecker [--type <plugin-type>] [--cloud <cloud-provider-name>]")
+    print("Usage: labchecker [--type <plugin-type>] [--cloud <cloud-provider-name>] [--nocolor]")
     print("optional arguments:")
     print("\t--help\t\tShow help message and exit.")
     print("\t--type\t\tSpecify a plugin type, like 'provider', 'all'(default)")
```

Comment:
please update help message, type is `provider` or `jobs` or `nodepool` now, right? what's the different with provider and jobs?

Reply:
Current, only `nodepool` and `jobs` is available.
- `nodepool` is for checking the nodepool side status, like the provider status check, nodepool configuration check
- `jobs` is for checking the zuul-jobs, or some custom jobs requirement.

and the description is out-of-date, I will update in next ps